src/utilities/api_calls.js

```diff
@@ -0,0 +1,28 @@
+var rp = require('request-promise');
+
+var post_api = function (post_uri, post_body) {
+	var options = {
+	    method: 'POST',
+	    uri: post_uri,
+	    body: post_body,
+	    json: true // Automatically stringifies the body to JSON	
+	};
+	rp(options)
+	    .then(function (parsedBody) {
+	        console.log('post succeeded');
```

Comment:
It's possible this succeeds regardless of whether the call succeeds. Please ensure that you are getting the actual data on the response.